rust/src/detect/uint.rs

```diff
@@ -101,9 +102,11 @@ fn parse_uint_subslice(parts: &[&str]) -> Option<(i32, i32)> {
     }
     let (_, (start, end)) = parse_uint_subslice_aux(parts[2]).ok()?;
```

Comment:
@jasonish is there a rust way to log a message and return the error ?

Reply:
Nothing special, catch the error, log it, pass it up?